### PR TITLE
Update intro material

### PR DESCRIPTION
**Pull request by arthurevans**
In this PR:

- Site updates based on Eric's feedback.
- Removed the landing page warning about updating the bower.json file (no longer required with the new name).
- Added notes on the change watcher argument order.
- Added graph for benchmarks.

### CODE REVIEW

**Comment by arthurevans** [comment]
@ebidel PTAL

File: 0.8/docs/devguide/feature-overview.md [hunk 1]
```diff
@@ -22,16 +22,47 @@ to make it easier and faster to make custom elements that work like standard DOM
...
+      Polymer({
+        is: "element-name",
+
+        // add properties and methods on the element's prototype
+
+        properties: {
+          // declare properties for the element's public API
+          greeting: {
+            type: String,
+            default: "Hello!"
```
Comment (ebidel) [inline 0.8/docs/devguide/feature-overview.md]: `value` right?

File: 0.8/docs/devguide/properties.md [hunk 2]
```diff
@@ -135,22 +136,27 @@ for more information.
...
-dashes.
+properties back to attributes, {{site.project_title}} maps attribute names to property
+names and the reverse. 
+
+When mapping attribute names to property names:
 
-When mapping attribute names to property names, the general rule is that
-attribute names are converted to lowercase (since the DOM is case-insensitive
-for attribute names).
+*   Attribute names are converted to lowercase property names. For example,
```
Comment (ebidel) [inline 0.8/docs/devguide/properties.md]: So it's not a requirement that attribute names contain a dash if their name contains more than one word? I thought in my testing this was the vase. Not so?

Comment (arthurevans) [inline 0.8/docs/devguide/properties.md]: Hm. I was trying to make this clearer. Here. the _attribute_ name is firstName, and it gets converted to the property name "firstname". (No camel case).

If you want camel case, you need to use a dash in the attribute name. Let me know if you can think of a clearer way to say this.

Comment (ebidel) [inline 0.8/docs/devguide/properties.md]: > If you want camel case, you need to use a dash in the attribute name. Let me know if you can think of a clearer way to say this.

This is the key bit I glossed over

**Comment by ebidel** [comment]
LGTM

**Pull request merged**